packages/@react-spectrum/button/src/Button.tsx

```diff
@@ -119,9 +125,8 @@ function Button<T extends ElementType = 'button'>(props: SpectrumButtonProps<T>,
         data-style={style}
         data-static-color={staticColor || undefined}
         aria-disabled={isPending ? 'true' : undefined}
-        aria-label={isPending ? undefined : buttonProps['aria-label']}
-        aria-labelledby={isPending ? undefined : buttonProps['aria-labelledby']}
-        aria-live={isPending && isFocused ? 'polite' : undefined}
+        aria-label={isPending ? isPendingAriaLiveLabel : buttonProps['aria-label']}
+        aria-labelledby={isPending ? isPendingAriaLiveLabelledby : buttonProps['aria-labelledby']}
```

Comment:
Using `aria-label={isPendingAriaLiveLabel}` and `aria-labelledby={isPendingAriaLiveLabelledby}` on the button element ensures that "pending" is included in the button's accessibility name when `isPending` is `true`.